### ISSUE TITLE
[EN][Quest] Ammo Kerblammo

### ISSUE THREAD

**Issue by morgast2** [Atlantiss/BugTracker#7651]
Hello,
in this quest you have to destroy ammo, the quest works, but the ammo are mostly spawn under terrain, so you see stars around but cant click on it.
Thanks for fix.

**Comment by Aival94** [comment]
Give quest link next time and a video might help :)

Wowhead Link: [Ammo Kerblammo](http://www.wowhead.com/quest=14042/ammo-kerblammo)
How it should work: [Ammo Kerblammo](https://www.youtube.com/watch?v=yERpoeo_eQw)

**Comment by morgast2** [comment]
Ok, thanks. problem is, that this
http://www.wowhead.com/quest=14042/ammo-kerblammo#screenshots:id=173016
are spawned under terrain.

**Comment by Axtron13** [comment]
You have to wait for it to respawn i've done this quest

**Comment by morgast2** [comment]
Yeah, you can finish that quest, but there are too many ammo spawned under texture. There are 3-4 spawned normaly and other 6 spawned under texture - you see just stars (symbol of quest item).
